supportedBackends/google/batch/src/main/scala/cromwell/backend/google/batch/api/GcpBatchRequestFactoryImpl.scala

```diff
@@ -243,10 +243,7 @@ class GcpBatchRequestFactoryImpl()(implicit gcsTransferConfiguration: GcsTransfe
       case GcpBatchLogsPolicy.CloudLogging =>
         LogsPolicy.newBuilder.setDestination(Destination.CLOUD_LOGGING).build
       case GcpBatchLogsPolicy.Path =>
-        LogsPolicy.newBuilder
-          .setDestination(Destination.PATH)
-          .setLogsPath(data.gcpBatchParameters.logfile.toString)
-          .build
+        ???
```

Comment:
Also rather than being config-driven, we probably want to be able to drive Cloud Logging from workflow options so Terra users can click a checkbox to get this behavior iff they want it.